src/views/authenticated/List.tsx

```diff
@@ -14,9 +14,11 @@ export function List({ data: unfilteredListItems, listPath }: Props) {
 	const [searchTerm, setSearchTerm] = useState<string>("");
 
 	const filteredListItems = useMemo(() => {
-		return unfilteredListItems.filter((item) =>
-			item.name.toLowerCase().includes(searchTerm.toLowerCase()),
-		);
+		return unfilteredListItems
+			.filter((item) =>
+				item.name.toLowerCase().includes(searchTerm.toLowerCase()),
+			)
+			.sort(comparePurchaseUrgency);
```

Comment:
PRAISE: I like the use of .sort() here.

Reply:
I agree! Great detail to implement the sorting even within the filtered list.

Reply:
I gotta pick your guys' brain a bit tomorrow about the logic of this piece of code :)